package.json

```diff
@@ -140,7 +140,8 @@
 		]
 	},
 	"cypress-cucumber-preprocessor": {
-		"nonGlobalStepDefinitions": true
+		"nonGlobalStepDefinitions": true,
+		"stepDefinitions": "cypress/e2e"
```

Comment:
can you add another line here to specify common specs
 "commonPath":"cypress/e2e/common"